<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="laPassword">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Passwort</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pbConnect">
        <property name="text">
         <string>Verbinden</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbDisconnect">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Trennen</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="laStatus">
        <property name="text">
         <string>getrennt</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="laServer">
      <property name="text">
       <string>Server</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="leDrivers">
      <property name="text">
       <string>Treiber</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="3">
     <widget class="QPushButton" name="pbExit">
      <property name="text">
       <string>Verlassen</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>200</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="laDatabase">
      <property name="text">
       <string>Datenbank</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="leUser"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="leServer"/>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="leDatabase"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="laUser">
      <property name="text">
       <string>Benutzername</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="lePassword">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="sbDrivers"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>sbDrivers</tabstop>
  <tabstop>leServer</tabstop>
  <tabstop>leDatabase</tabstop>
  <tabstop>leUser</tabstop>
  <tabstop>lePassword</tabstop>
  <tabstop>pbConnect</tabstop>
  <tabstop>pbDisconnect</tabstop>
  <tabstop>pbExit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pbExit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
